File: auth-service/src/main/java/com/jaksimsam1/authservice/domain/auth/repository/AuthRepository.java
package com.jaksimsam1.authservice.domain.auth.repository;

import com.jaksimsam1.authservice.domain.auth.repository.r2dbc.AuthReactiveRepository;
import com.jaksimsam1.authservice.domain.auth.repository.query.AuthQueryRepository;

public interface AuthRepository extends AuthReactiveRepository, AuthQueryRepository {
}

File: auth-service/src/main/kotlin/com/jaksimsam1/authservice/domain/auth/repository/AuthRepository.kt
package com.jaksimsam1.authservice.domain.auth.repository

import com.jaksimsam1.authservice.domain.auth.repository.jdsl.AuthJdslRepository
import com.jaksimsam1.authservice.domain.auth.repository.jpa.AuthJpaRepository

interface AuthRepository: AuthJpaRepository, AuthJdslRepository {
}